daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -6,6 +6,7 @@ package btrfs
 #include <stdlib.h>
 #include <dirent.h>
 #include <btrfs/ioctl.h>
+#include <btrfs/ctree.h>
```

Comment:
I don't even know if we need that I ended up cleaning up a bunch of the code in a second commit, but will def check ;)